### PR TITLE
feat: Rename verificationMethodId to kid [DEV-3670]

### PR DESCRIPTION
**Pull request by lampkin-diet**
- Rename verificationMethoID to kid
- Add test for DID creation
- Bump other deps

### CODE REVIEW

**Comment by ankurdotb** [comment]
Task linked: [DEV-3670 DID-Registrar: Move to kid from verificationMethodId](https://app.clickup.com/t/6600954/DEV-3670)

File: src/helpers/helpers.ts [hunk 1]
```diff
@@ -9,7 +9,7 @@ import { ISignInfo } from '../types/types';
 export function convertToSignInfo(payload: ISignInfo[]): SignInfo[] {
 	return payload.map((value) => {
 		return {
-			verificationMethodId: value.verificationMethodId,
+			verificationMethodId: value.kid,
```
Comment (DaevMithran) [inline src/helpers/helpers.ts:12]: The key here should be change from `verificationMethodId` to `kid` right?

**Review by DaevMithran — CHANGES_REQUESTED** [review]

File: src/helpers/helpers.ts (hunk 1 above)
Comment (DaevMithran) [inline src/helpers/helpers.ts:12]:
```suggestion
			kid: value.kid,
```

**Review by DaevMithran — CHANGES_REQUESTED** [review]

**Review by Eengineer1 — DISMISSED** [review]
Looks good.

Approved.

**Pull request merged**